### PR TITLE
Update ACS GA Java Releases

### PR DESCRIPTION
**Pull request by jorgegarchirota**
@tophpalmer , @ankitarorabit , @RezaJooyandeh  please take a look to the Java release notes.

### CODE REVIEW

**Comment by RezaJooyandeh** [comment]
Will this update the blog post for March? I assumed these changes would go to the April so they show up in the next Azure SDK blog post.

**Comment by jorgegarchirota** [comment]
Yes it will, Reza. The entry blog will be on April (first week)

**Comment by jorgegarchirota** [comment]
@g2vinay  is any update or comment on this? Thank you!

File: releases/2021-03/java.md [hunk 1]
```diff
@@ -728,6 +751,26 @@ This is the initial release of Azure Device Update for IoT Hub library.
 
 - Azure Resource Manager AzureDigitalTwins client library for Java. This package contains Microsoft Azure SDK for AzureDigitalTwins Management SDK. Azure Digital Twins Client for managing DigitalTwinsInstance. Package tag package-2020-12. For documentation on how to use this package, please see [Azure Management Libraries for Java](https://aka.ms/azsdk/java/mgmt).
 
+### Azure Communication Chat 1.0.0 [Changelog](https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/communication/azure-communication-chat/CHANGELOG.md#100-2021-03-29)
+**Includes all changes from 1.0.0-beta.1 to  1.0.0.beta.5**
+
+#### Breaking Changes
```
Comment (g2vinay) [inline releases/2021-03/java.md:757]: This section can be removed if possible, if not then summarize it to one or two bullet points.

File: releases/2021-03/java.md [hunk 2]
```diff
@@ -459,13 +464,19 @@ To use the GA and beta libraries, refer to the Maven dependency information belo
...
   <groupId>com.azure</groupId>
   <artifactId>azure-communication-common</artifactId>
-  <version>1.0.0-beta.5</version>
+  <version>1.0.0-beta.6</version>
+</dependency>
+
+<dependency>
+  <groupId>com.azure</groupId>
+  <artifactId>azure-communication-identity</artifactId>
+  <version>1.0.0</version>
```
Comment (g2vinay) [inline releases/2021-03/java.md:479]: this should be moved up to group with GA packages.

File: releases/2021-03/java.md [hunk 3]
```diff
@@ -459,13 +464,19 @@ To use the GA and beta libraries, refer to the Maven dependency information belo
 <dependency>
   <groupId>com.azure</groupId>
   <artifactId>azure-communication-common</artifactId>
-  <version>1.0.0-beta.6</version>
+  <version>1.0.0</version>
```
Comment (g2vinay) [inline releases/2021-03/java.md:467]: move up to group with GA packages.

File: releases/2021-03/java.md [hunk 4]
```diff
@@ -477,7 +488,13 @@ To use the GA and beta libraries, refer to the Maven dependency information belo
...
   <groupId>com.azure</groupId>
   <artifactId>azure-communication-phonenumbers</artifactId>
-  <version>1.0.0-beta.6</version>
+  <version>1.0.0-beta.7</version>
+</dependency>
+
+<dependency>
+  <groupId>com.azure</groupId>
+  <artifactId>azure-communication-sms</artifactId>
+  <version>1.0.0</version>
```
Comment (g2vinay) [inline releases/2021-03/java.md:497]: move up to group with GA packages.

**Review by g2vinay — APPROVED** [review]

**Comment by jorgegarchirota** [comment]
These release notes will be on April

**Pull request closed without merging**